From 92090b92fab207250d5b8d5a4a36aa34f5a91f19 Mon Sep 17 00:00:00 2001
From: Weijie Gao <weijie.gao@mediatek.com>
Date: Fri, 17 Jan 2025 17:16:33 +0800
Subject: [PATCH 01/15] board: mediatek: mt7622: remove board_late_init

The function board_late_init defined for mt7622 is useless now. Just
remove it.

Signed-off-by: Weijie Gao <weijie.gao@mediatek.com>
---
 board/mediatek/mt7622/mt7622_rfb.c | 7 -------
 1 file changed, 7 deletions(-)

--- a/board/mediatek/mt7622/mt7622_rfb.c
+++ b/board/mediatek/mt7622/mt7622_rfb.c
@@ -15,10 +15,3 @@ int board_init(void)
 {
 	return 0;
 }
-
-int board_late_init(void)
-{
-	gd->env_valid = 1; //to load environment variable from persistent store
-	env_relocate();
-	return 0;
-}
